test/library/packages/Zarr/ZarrTest.chpl

```diff
@@ -185,7 +185,7 @@ proc updateChunkTest(type dtype) {
 proc main() {
   testGetLocalChunks();
   testUndistributedArray();
-  const typeTuple = (0:int(32), 0:int(64), 0:real(32), 0:real(64));
+  const typeTuple = (0:int(64), 0:real(32));
```

Comment:
Why remove them entirely, why not just separate them into another test file?

Reply:
I can def break it into multiple test files; I'll probably break by testcase rather than type.

Reply:
Works for me :)